commons-math-legacy-core/src/main/java/org/apache/commons/math4/legacy/core/dfp/Dfp.java

```diff
@@ -663,9 +663,7 @@ public Dfp getTwo() {
     /** Shift the mantissa left, and adjust the exponent to compensate.
      */
     protected void shiftLeft() {
-        for (int i = mant.length - 1; i > 0; i--) {
-            mant[i] = mant[i - 1];
-        }
+        if (mant.length - 1 >= 0) System.arraycopy(mant, 0, mant, 1, mant.length - 1);
```

Comment:
use `> 0` as arraycopy with a length of 0 is a wasted call to a native method. Add enclosing braces to fix checkstyle.